solidity/contracts/utility/TokenHolder.sol

```diff
@@ -35,6 +37,11 @@ contract TokenHolder is ITokenHolder, Owned, Utils {
         address _to,
         uint256 _amount
     ) public virtual override ownerOnly validAddress(address(_token)) validAddress(_to) notThis(_to) {
```

Comment:
Just an FYI, but we have a `validExternalAddress` modifier that combines both of these checks together

Reply:
We don't need the 'notThis' check IMO, it's a pure waste of gas (and bytecode size).

Reply:
In this case, 100%